src/drawer/index.unit.tsx

```diff
@@ -8,47 +8,48 @@ const defaultProps = {
 describe('Drawer', () => {
   it('Renders with a custom className', () => {
     const customClassName = 'custom-drawer'
-    const wrapper = shallow(<Drawer {...defaultProps} className={customClassName}>body</Drawer>)
+    const wrapper = shallow(
+      <Drawer {...defaultProps} className={customClassName}>
+        body
+      </Drawer>,
+    )
     expect(wrapper.hasClass(customClassName)).toBe(true)
   })
 
   it('Should call `onOpen` and `scrollLock` after open', () => {
     const onOpen = jest.fn()
-    const onChange = jest.fn()
+    const onTransitionEnd = jest.fn()
     const wrapper = mount(
```

Comment:
Remove the usage of mount, use shallow instead

Reply:
done